CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - [core] Add a way to prevent application exit from extensions.
 - [core] Prevent application exit if some editors are dirty.
 - [languages] Add a preference for every language contribution to be able to trace the communication client <-> server
+- [plug-in] added `tasks.onDidStartTask` Plug-in API
```

Comment:
added -> Add (capital letter and present tense)

Reply:
@simark I fixed your comment and updated (rebased) my branch, can I merge it?

Reply:
Oops.  Based on https://github.com/theia-ide/theia/pull/3897, it seems like you may have to go back to what you had originally :).  We'll see what we decide in the end.

Reply:
changed to original condition

Reply:
@simark could you review - can I merge my PR?